src/mbgl/gfx/vertex_vector.hpp

```diff
@@ -7,16 +7,24 @@
 namespace mbgl {
 namespace gfx {
 
+struct VertexVectorBase {
```

Comment:
This should ether be a class type or the forward decl in `vertex_attribute.hpp` needs to change to struct